config/keys.go

```diff
@@ -4,10 +4,8 @@ package config
 type ContextKey string
 
 const (
-	// DAOService is the key to store the dao service
-	DAOService = ContextKey("DAOService")
```

Comment:
Reasoning for removing this? Is it all tied up in the changes to passing in middleware to create_payable_resource.go?

Reply:
Okay, always weary when removing something even if it's flagged as not used :)...let's see how it goes, did wonder whether there was a dependency on it somehow in external app (api-core maybe) but don't think there is after looking about.

Reply:
I had a look to see if it was used but couldn't find anything either. Also using this as an opportunity to tidy up the code.